oioioi/contests/templatetags/get_user_name.py

```diff
@@ -51,6 +53,13 @@ def _get_user_name(self, context, user):
 class FullNameUserInfoLinkNode(UserInfoLinkNode):
     def _get_user_name(self, context, user):
         return get_user_display_name(user)
+    
+
+class NamePermissionsUserInfoLinkNode(UserInfoLinkNode):
+    def _get_user_name(self, context, user):
+        if is_forum_moderator(context['request'], user):
+            return get_user_display_name(user) + _(" (moderator)")
```

Comment:
I am thinking if one could have a display name with (moderator) probably by modifying their last name. Regardless of the possibility, could we make the badge graphically different from the `display_name`? I think we should avoid some "social engineering" attacks if possible.

Reply:
I was thinking about the same thing in regards of safety. It required me to use two separate TemplateTags instead of handling both name and permissions in one. Now the user's permissions are displayed in a Bootstrap badge.